### ISSUE TITLE
feature request: inline window workflow

### ISSUE THREAD

**Issue by kmoschcau** [ms-jpq/chadtree#298]
Hi, I'd love to use this over NERDTree, but it is missing a few small features that make this kinda hard with my current workflow.

Basically what's needed is the following:

- [ ] allow hijacking of the open commands on directories to open CHADtree in the window where the command was run, opening the directory that was specified to the command
- [ ] allow to open files in the same window as CHADtree when using the open commands

Basically this allows a workflow of me just picking whatever window layout that I want, quickly open the file explorer in a window I already have open and then just open a file in the same window. In that workflow there is also no need to worry about in which window a file will be opened, because you determine that simply by where you open CHADtree.
